modules/recommended-node-host-practices.adoc

```diff
@@ -27,6 +27,13 @@ actual container starting. Therefore, a system running 10 pods will actually
 have 20 containers running.
 ====
 
+[NOTE]
+====
+Disk IOPS throttling from the cloud provider might have an impact on CRI-O and kubelet. 
+They might get overloaded when there are large number of I/O intensive pods running on 
+the nodes. It is recommended that you use volumes with higher throughput.
```

Comment:
This is pretty generic.   If we are not going to recommend a specific throughput, should we say "It is recommended that you monitor disk I/O on nodes and use volumes with sufficient through put for the workload."

I don't think it is critical, but sounds a bit more specific.

Reply:
Makes sense @mffiedler, will modify it.

Reply:
Fixed it, thanks.